content/en/continuous_integration/troubleshooting.md

```diff
@@ -31,8 +31,12 @@ This page provides information to help you troubleshot issues with CI Visibility
 4. If you still don't see any results, [contact Support][2] for troubleshooting help.
 
 ### You are uploading JUnit test reports with `datadog-ci` but some or all tests are missing
-If you are uploading JUnit test report files with `datadog-ci` CLI, make sure that the timestamp of the reported tests is not older than **18 hours** before the moment the report is uploaded.
+If you are uploading JUnit test report files with `datadog-ci` CLI and you do not see the tests, it is likely that is being discarded because the report is considered incorrect.
```

Comment:
```suggestion
If you are uploading JUnit test report files with `datadog-ci` CLI and you do not see the tests, it is likely the tests are being discarded because the report is considered incorrect.
```